packages/regenesis-surgery/test/unit-tests/handlers.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { KECCAK256_RLP_S, KECCAK256_NULL_S } from 'ethereumjs-util'
+
+/* Imports: Internal */
+import { expect } from '../setup'
+import { handlers } from '../../scripts/handlers'
+import { Account, AccountType } from '../../scripts/types'
+
+describe('Handlers', () => {
+  const dummyAccount: Account = {
+    address: '0x0000000000000000000000000000000000000420',
+    nonce: 69,
+    balance: '0',
+    codeHash: '420e69',
+    root: '420e69',
+    code: '608060405',
+    storage: {
+      '0x0000000000000000000000000000000000000420':
+        '0000000000000000000000000000000000000420',
+    },
+  }
+
```

Comment:
Good job so fat. For eoa, it would also be useful to test for the root/codehash being the expected values